xpsi/tests/test_interstellar.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import pytest
+from xpsi.Interstellar import Interstellar
+
+class TestInterstellar:
+
+    class MockInterstellar(Interstellar):
+        def attenuation(self, energies):
+            return np.exp(-energies)  # Mock implementation for testing
+
+    def setup_method(self):
+        self.model = self.MockInterstellar()
```

Comment:
Unless something has changed in PyTest (and it might well have), I'd expect there to be a method called `setup_class` and anything you'll want to use later on to be stored in `cls.model`.